CHANGELOG.md

```diff
@@ -22,6 +24,7 @@
 
 ## Bug fixes
 
+-   Improved flexibility of parameter values so that parameters can be set as functions or scalars ([#723](https://github.com/pybamm-team/PyBaMM/pull/723))
```

Comment:
does this work for things like diffusivity which we assume is a function of some of the variables, or just functions of time/parameters?

Reply:
It should work with both. If something is assumed to be a function in the model then we can pass a scalar and it will always take that scalar value without changing the model. But the other way round (scalar to function) doesn't (can't) work, as you have to tell the model what the function depends on